src/components/radio-button/radio-button-group/radio-button-group.ts

```diff
@@ -304,18 +304,9 @@ export class SbbRadioButtonGroup extends LitElement {
       return;
     }
 
-    let current: number;
-    let nextIndex: number;
-
-    if (this._hasSelectionPanel) {
-      current = enabledRadios.findIndex((e: SbbRadioButton) => e === evt.target);
-      nextIndex = getNextElementIndex(evt, current, enabledRadios.length);
-    } else {
-      const checked: number = enabledRadios.findIndex((radio: SbbRadioButton) => radio.checked);
-      nextIndex = getNextElementIndex(evt, checked, enabledRadios.length);
-      enabledRadios[nextIndex].select();
-    }
-
+    const current = enabledRadios.findIndex((e: SbbRadioButton) => e === evt.target);
```

Comment:
In case it is a selection panel which has some content, it's not allowed to directly set selection for accessibility reasons. So what can be improved is, enhancing the check for this._hasSelectionPanel with a check for if the selection panel has also some content. As a result, direct selection would only happen, if a selection panel without a content is present.